src/Libraries/RevitNodes/Elements/Views/View.cs

```diff
@@ -213,5 +211,33 @@ public Revit.Filter.OverrideGraphicSettings FilterOverrides(Revit.Filter.Paramet
         }
 
         #endregion
+
+        #region Scale
+
+        /// <summary>
+        /// Set View Scale
+        /// </summary>
+        /// <param name="scale">View scale is the ration of true model size to paper size.</param>
+        /// <returns name="View">View</returns>
+        public Element SetScale(int scale=100)
+        {
+            if (Autodesk.Revit.DB.View.IsValidViewScale(scale))
+            {
+                if (this.InternalView.Scale != scale)
```

Comment:
is scale a double? if so use an epsilon value tolerance value to compare them please.